prime-router/src/main/kotlin/ActionLog.kt

```diff
@@ -50,7 +50,7 @@ data class ActionLog(
     var reportId: UUID? = null,
     var action: Action? = null,
     val type: ActionLogLevel = ActionLogLevel.info,
-    val created_at: Instant = Instant.now(),
+    val createdAt: Instant = Instant.now(),
```

Comment:
Changed is not related to the ticket, but fixes a spurious error that can occur while inserting `ActionLog`